SRC/NewPortfolio/ContactPage/contact.css

```diff
@@ -0,0 +1,83 @@
+body {
+  margin: 0;
+  padding: 0;
+  text-align: center;
+  background: url(./cloudPoster.jpg);
+  background-size: cover;
+  font-family: 'Roboto Mono', monospace;
+}
+nav {
```

Comment:
consider adding: 
`.main-container {`
`height: 100%;`
`background-color: rgba(255, 255, 255, 0.25);`
`}`
it's a layer over the background image and it'll help your text pop out and not get lost in the background image. play with the 0.25. add more or less. whatever you like!